src/pages/workspaces/workspace/Analyses.js

```diff
@@ -145,70 +138,49 @@ const AnalysisCard = ({ namespace, name, updated, metadata, toolLabel, listView,
-    img({ src: appIconSrc, style: { height: 40, width: 40 } })
+  const toolContainer = div({ style: { display: 'flex', flex: 1, flexDirection: 'row', alignItems: 'center' } }, [
+    toolIcon,
+    // this is the tool name, i.e. 'Jupyter'. It is named identical to the header row to simplify the sorting code at the cost of naming consistency.
+    application
   ])
 
   return a({
     href: analysisLink,
-    style: {
-      ...Style.elements.card.container,
-      ...analysisCardCommonStyles(listView),
-      flexShrink: 0
-    }
-  }, listView ? [
-    appIcon,
-    toolLabel,
-    title,
-    div({ style: { flexGrow: 1 } }),
-    locked && h(Clickable, {
-      style: { display: 'flex', paddingRight: '1rem', color: colors.dark(0.75) },
-      tooltip: `This analysis is currently being edited by ${lockedBy || 'another user'}`
-    }, [icon('lock')]),
-    h(TooltipTrigger, { content: Utils.makeCompleteDate(updated) }, [
-      div({ style: { fontSize: '0.8rem', marginRight: '0.5rem' } },
-        `Last edited: ${Utils.makePrettyDate(updated)}`)
-    ]),
-    analysisMenu
-  ] : [
-    div({ style: { display: 'flex' } }, [
-      title,
-      div({ style: { flexGrow: 1 } }),
-      locked && h(Clickable, {
-        style: { display: 'flex', padding: '1rem', color: colors.dark(0.75) },
-        tooltip: `This analysis is currently being edited by ${lockedBy || 'another user'}`
-      }, [icon('lock')])
-    ]),
-    div({
-      style: {
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'space-between',
-        borderTop: `solid 1px ${colors.dark(0.4)}`,
-        paddingLeft: '0.5rem', paddingRight: '0.5rem', height: '2.5rem',
-        backgroundColor: colors.light(0.4),
-        borderRadius: '0 0 5px 5px'
-      }
-    }, [
-      h(TooltipTrigger, { content: Utils.makeCompleteDate(updated) }, [
-        div({ style: { fontSize: '0.8rem', marginRight: '0.5rem' } }, [
-          'Last edited: ',
-          Utils.makePrettyDate(updated)
+    style: _.merge({
+      ...Style.cardList.longCardShadowless
+    }, { marginBottom: '.75rem', paddingLeft: '1.5rem' })
+  }, [
+    toolContainer,
+    artefactName,
+    //The 10% allows for the longest date possible (September XX, XXXX), plus a locked symbol, without text wrapping when the screen is maximized
+    div({ style: { flex: '0 0 10%', display: 'flex', flexDirection: 'row', justifyContent: 'flex-left' } }, [
+      div({ style: { flex: 1, display: 'flex' } }, [
+        locked && h(Clickable, {
+          style: { display: 'flex', paddingRight: '1rem', color: colors.dark(0.75) },
+          tooltip: `This analysis is currently being edited by ${lockedBy || 'another user'}`
+        }, [icon('lock')]),
+        h(TooltipTrigger, { content: Utils.makeCompleteDate(lastModified) }, [
+          div({ style: { fontSize: '0.8rem', display: 'flex', alignItems: 'center' } }, [Utils.makePrettyDate(lastModified)])
```

Comment:
```suggestion
    div({ style: { display: 'flex', justifyContent: 'flex-end' } }, [
      locked && h(Clickable, {
        style: { display: 'flex', paddingRight: '1rem', color: colors.dark(0.75) },
        tooltip: `This analysis is currently being edited by ${lockedBy || 'another user'}`
      }, [icon('lock')]),
      h(TooltipTrigger, { content: Utils.makeCompleteDate(lastModified) }, [
        div({ style: { fontSize: '0.8rem', display: 'flex', alignItems: 'center' } }, ['September 9, 2021'])
      ]),
```

I was still seeing some wrapping on the date, depending on screen width. I believe that suggestion takes care of it.

Reply:
I addressed this with the previous wrapping comment. Essentially, no-wrap in combination with some other styling